tv.plex.PlexDesktop.yml

```diff
@@ -0,0 +1,117 @@
+app-id: tv.plex.PlexDesktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Plex
+finish-args:
+  - --socket=x11
```

Comment:
I assume this still has the same wayland issues as the HTPC version.

Reply:
Correct. Both will get the fix at the same time (they share a lot of code).